.cirrus.yml

```diff
@@ -75,9 +75,37 @@ testing_task:
         failure_journal_log_script: '${_JOURNALCMD} || true'
 
 
+# This task runs `make vendor` followed by ./hack/tree_status.sh to check
+# whether the git tree is clean.  The reasoning for that is to make sure
+# that the vendor.conf, the code and the vendored packages in ./vendor are
+# in sync at all times.
+vendor_task:
+
+    only_if: $CIRRUS_CHANGE_MESSAGE !=~ '.*\*\*\*\s*CIRRUS:\s*TEST\s*IMAGES\s*\*\*\*.*'
+
+    env:
+        CIRRUS_WORKING_DIR: "/var/tmp/go/src/github.com/containers/buildah"
+        GOPATH: "/go"
+        GOSRC: "/go/src/github.com/containers/buildah"
+
+    # Runs within Cirrus's "community cluster"
+    container:
+        image: docker.io/library/golang:1.13
+        cpu: 1
+        memory: 1
```

Comment:
If this task runs too slowly, there's no harm in bumping CPU and Memory upward.  I believe the default is 8 and 12, so there's quite a bit of headroom available.